docling_core/types/doc/base.py

```diff
@@ -12,6 +12,7 @@ class ImageRefMode(str, Enum):
 
     PLACEHOLDER = "placeholder"
     EMBEDDED = "embedded"
+    EXPORTED_TO_PNG = "exported_to_png"
```

Comment:
I don't think `EXPORTED_TO_PNG` is a good name, this must be a generic term for externally referenced files. It should be equally valid to put a web URI or other protocol for this case, and the image format must not be part of it. Depending on what future backends give us it could also be used for a native image format in the source doc, e.g. SVG or other.

Proposals:
- `URI`
- `FILE_URI`